client/src/lib/dependency-injection/ApolloGraphqlManager.js

```diff
@@ -157,22 +199,31 @@ class ApolloGraphqlManager {
    * @param field
    * @returns {ApolloGraphqlManager}}
    */
-  addField(field) {
-    return this.addFields([field]);
+  addField(path = ROOT_FIELD, field) {
```

Comment:
Doc block needs to be updated with the new path argument (it also already has broken jsdoc syntax)